inductiva/_cli/cmd_storage/remove.py

```diff
@@ -1,20 +1,35 @@
 """Remove the user's remote storage contents via CLI."""
 
+import sys
+
 from inductiva import storage
 
 
 def remove(args):
     """Remove user's remote storage contents."""
-    path = args.path
+    paths = args.path
     confirm = args.confirm
+    all_paths = args.all
+
+    if paths and all_paths:
+        print("inductiva storage remove: error: "
+              "argument path not allowed with argument --all")
+        sys.exit(1)
```

Comment:
errors should go to stderr; also, 2 `:` in the same sentence:

```suggestion
    if paths and all_paths:
        print("inductiva storage remove error:\n"
              "argument path not allowed with argument --all", file=sys.stderr)
        sys.exit(1)
```

Reply:
I copied the same text from the argparser error when we do
inductiva tasks list -n 5 -id sadase
We need to make these kinds of messages consistent. Should I change it on the list too?

Reply:
Yes, apply it in every place you consider necessary :D